docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,126 @@
+    - `Windows.UI.Xaml.Duration`-> `Microsoft.UI.Xaml.DurationHelper`
+    - `Windows.UI.Xaml.GridLength`-> `Microsoft.UI.Xaml.GridLengthHelper`
+    - `Windows.UI.Xaml.Thickness`-> `Microsoft.UI.Xaml.ThicknessHelper` ->
+    - `Windows.UI.Xaml.Controls.Primitives.GeneratorPosition` -> `Microsoft.UI.Xaml.Controls.Primitives.GeneratorPositionHelper`
+    - `Windows.UI.Xaml.Media.Matrix` -> `Microsoft.UI.Xaml.Media.MatrixHelper` 
+    - `Windows.UI.Xaml.Media.Animation.KeyTime` -> `Microsoft.UI.Xaml.Media.Animation.KeyTimeHelper`
+    - `Windows.UI.Xaml.Media.Animation.RepeatBehavior` -> `Microsoft.UI.Xaml.Media.Animation.RepeatBehaviorHelper`
+
+### What The Conversion Analyzers Do Not Do:
+
+ - Modify .csproj files or modify/resolve conflicting NuGet packages
+ - Remove deprecated Code
+ - Analyze/Convert .xaml files
+
+### Examples
+
+#### Namespaces Before Running Analyzers:
+```csharp
+using Windows.UI;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+```
+
+#### Namespaces After Running Analyzers:
+```csharp
+using Microsoft.UI;
+using Microsoft.UI.ViewManagement;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+```
+
+#### OnLaunched Method Before Running Analyzers:
+```c#
+protected override async void OnLaunched(LaunchActivatedEventArgs args)
+{
+    await EnsureWindow(args)
+}
+```
+
+#### OnLaunched Method After Running Analyzers:
+```csharp
+protected override async void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
+{
+    await EnsureWindow(args.UWPLaunchActivatedEventArgs)
+}
+```
+## Conversion Process
+**Note: `Microsoft.WinUI.Convert` NuGet Package is Currently Unavailable During Prerelease**
+
+Necessary steps for converting a WinUI C# App to WinUI3: 
+
+1. Install `Microsoft.WinUI` and `Microsoft.WinUI.Convert` NuGet packages in your app using the NuGet package manager: see the [Install WinUI 3 Preview](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#install-winui-3-preview-2) page for more information.
+    
+    ![Visual Studio Lightbulb Suggestion](./images/newPackage.png#thumb)
+
+2. Uninstall Microsoft.UI.Xaml from your solution. Additional conflicting packages such as `Microsoft.Xaml.Behaviors.*` may also need to be removed. 
+    
+    ![Visual Studio Lightbulb Suggestion](./images/uninstallOldPackage.png#thumb)
+
+3. Light Bulb suggestions should highlight issues that need to be updated for WinUI3 conversion. 
+
+    ![Visual Studio Lightbulb Suggestion](./images/lightbulb.png#thumb)
+
+4. Click the down arrow by the lightbulb, Convert to WinUI3 and select Fix all occurences in project. 
+
+    ![Visual Studio Lightbulb Suggestion](./images/analyzer-1.png#thumb)
+
+5. All namespace changes should be fixed in your project!
+
+6. The Analyzer can only parse C# code and not Xaml. WebView is now WebView2 and should be updated: see the [Getting Started](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) page for more information.
```

Comment:
The tool can't do any code updates for WebView2?

Reply:
I have not found a way to use Roslyn to modify additional files. We may be able to address this with try-convert using a find/replace.